snapshot-labs/snapshot.js
# fix: Increase choices limit for turbo

## Summary

closes https://github.com/snapshot-labs/snapshot.js/issues/1042

- New limits for turbo: 1000, default: 500
- Add tests to check new limits

## How to test

- run `yarn test`
- change `"turbo": 500`
- see error while passing more than 500 for turbo space
